vuepress/docs/v6.3.2/docs/getting-started/README.md

```diff
@@ -146,7 +146,7 @@ To install Entando, we'll add `Custom Resources`, create a `Namespace`, download
 #### Create Namespace
 
 ::: tip What are Namespaces?
-Kubernetes supports multiple virtual clusters backed by the same physical cluster. These virtual clusters are called [namespaces.](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+Kubernetes supports multiple virtual clusters backed by the same physical cluster. These virtual clusters are called [namespaces](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
```

Comment:
Minor, probably still need a period when there's more than one sentence. Otherwise it looks truncated/unfinished.

Reply:
sorry - i thought that was what i was correcting! this is the only link instance where the period is included within the brackets; all others place the period after the parentheses. should i change all [](). to [.]()?

Reply:
<img width="567" alt="Screen Shot 2021-11-14 at 12 31 06 PM" src="https://user-images.githubusercontent.com/93157274/141697559-86d19e9d-28a4-4cca-b38e-bab1e015668b.png">

Reply:
<img width="603" alt="Screen Shot 2021-11-14 at 12 31 30 PM" src="https://user-images.githubusercontent.com/93157274/141697577-dca79b93-441c-4f4e-a932-387e7c5cbe28.png">

Reply:
should the position of the . in the above be changed?

Reply:
I think I read the diff wrong at first, sorry. It looks a little strange with the period after the link but since it's not actually part of the link I think this is probably the best we can do. Let's keep it like you have it.

Reply:
i'm not happy with it either, but agreed